.travis.yml

```diff
@@ -7,3 +7,4 @@ before_script:
 
 script:
   - npm run lint
+  - npm test
```

Comment:
Note that, by default, Travis does the following:

```
before_install:
  - npm install

script:
  - npm test
```

Linting should run every time tests are run (see comments on `package.json`).